great_expectations/core/result_format.py

```diff
@@ -3,8 +3,26 @@
 import enum
 from typing import Final, Literal, Union
 
+from great_expectations._docs_decorators import public_api
 
+
+@public_api
 class ResultFormat(str, enum.Enum):
+    """
+    Responsible for configuring the level of detail for Validation Results in Data Docs.
```

Comment:
Referenced in docs by other signatures but we have links - we should ensure that any classes referenced by other classes/methods have their own page